net-analyzer/gvm/gvm-21.4.0.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit optfeature
+
+DESCRIPTION="Greenbone Vulnerability Management, previously named OpenVAS"
+HOMEPAGE="https://www.greenbone.net/en/"
+
+SLOT="0"
+LICENSE="GPL-2"
+KEYWORDS="~amd64 ~x86"
+IUSE="cli +cron +extras +gsa ldap ospd radius"
+
+RDEPEND="
+	>=net-analyzer/gvm-libs-${PV}[extras?,ldap?,radius?]
+	>=net-analyzer/gvmd-${PV}[extras?]
+	>=net-analyzer/openvas-scanner-${PV}[cron?,extras?]
+	cli? ( >=net-analyzer/gvm-tools-${PV} )
```

Comment:
gvm-tools seems to follow a different version scheme. Maybe

```suggestion
	cli? ( >=net-analyzer/gvm-tools-21.10.0 )
```

But that would be effictively what we have right now, so it appears it does not matter.